tests/unit/sync/transformTaxonomyGroupModel.test.ts

```diff
@@ -0,0 +1,77 @@
+import { describe, expect, it } from "@jest/globals";
+import { TaxonomyContracts } from "@kontent-ai/management-sdk";
+
+import { transformTaxonomyGroupsModel } from "../../../src/modules/sync/modelTransfomers/taxonomyGroups";
+
+const input: ReadonlyArray<TaxonomyContracts.ITaxonomyContract> = [
+  {
+    id: "taxonomyGroup!Id",
+    last_modified: "groupModified1",
+    name: "taxonomyGroup1",
+    codename: "taxonomy_group1",
+    external_id: "extId_taxonomy_group1",
+    terms: [
+      {
+        id: "nestedTermId1",
+        name: "nestedTerm",
+        codename: "nested_term",
+        last_modified: "nestedTermModofied",
+        terms: [],
+      },
+    ],
+  },
+  {
+    id: "taxonomyGroup2Id",
+    last_modified: "groupModified2",
+    name: "taxonomyGroup2",
+    codename: "taxonomy_group2",
+    external_id: "extId_taxonomy_group2",
+    terms: [
+      {
+        id: "nestedTermId2",
+        name: "nestedTerm2",
+        codename: "nested_term2",
+        last_modified: "nestedTermModofied2",
+        external_id: "extId_nested_term2",
+        terms: [],
+      },
+    ],
+  },
+];
+
+describe("transformTaxonomyGroupsModel", () => {
+  it("transformTaxonomyGroupsModel", () => {
+    const result = transformTaxonomyGroupsModel(input);
+
+    expect(result).toMatchInlineSnapshot(`
```

Comment:
I would prefer creating an adjusted clone of the `input` and check that the `result` equals the `expected` instead of a snapshot test. It would be clearer as to what changes are expected from the `transformTaxonomyGroupsModel`. You can create the `expected` like this:
```ts
const expected = [
  {
    ...input[0],
    id: undefined,
    last_modifed: undefined,
    //...
  },
  //...
];
```